Python/ceval.c

```diff
@@ -6919,7 +6922,6 @@ _PyEval_CallTracing(PyObject *func, PyObject *args)
     int save_tracing = tstate->tracing;
     int save_use_tracing = tstate->cframe->use_tracing;
     tstate->tracing = 0;
```

Comment:
Do we need to assert/ensure `tstate->cframe->use_tracing` is also 0?  (Presumably not, since all the tests are passing.)

Reply:
This is a C-API function, so we don't know what `use_tracing` will be.